doc/TSCC.rst

```diff
@@ -217,13 +323,10 @@ To figure out why a job is queued use :code:`scontrol show job <your_job_number>
 
 Debugging jobs the OS killed
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-#. Look at the output file :code:`<log_dir>/<jobName>.o<jobID>`, the first line should contain the node
-   name. (e.g. :code:`Nodes: tscc-5-7`)
-#. ssh into the node (you can do this to any node, but if you run a large process the OS will kill you because
-   you have not been scheduled to that node)
-#. Scan the os logs for a killed process `dmesg -T | grep <jobid>`
-
-The OS normally kills jobs because you ran over your memory limit.
+#. Look at the standard output and standard error files. Any error messages should be there.
+#. ssh into the node. You can do this to any node, but if you run a large process the OS will kill you because you have not been scheduled to that node. You can figure out the name of the node assigned to your job using :code:`squeue` once the status of the job is "RUNNING".
+#. Scan the os logs for a killed process :code:`dmesg -T | grep <jobid>`
```

Comment:
should we do this after ssh into the node? if the job is finished, is there anyway to know what node it was running on?

Reply:
ah, I think you can do it before or after ssh-ing into the node

and I think the best thing to do is to actually wait until the job is finished to run this
```suggestion
#. Scan the os logs for a job once it's been killed via :code:`dmesg -T | grep <jobid>`. You can get the jobid from :code:`squeue -u $USER`
```